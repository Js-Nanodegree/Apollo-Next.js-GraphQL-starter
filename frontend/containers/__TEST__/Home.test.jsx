import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import HomeContainer from '../Home';

describe('Home container', () => {
  it('should match snap shot', () => {
    const tree = renderer.create(<HomeContainer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
  it('should render child text', () => {
    const wrapper = shallow(<HomeContainer />);

    expect(wrapper.find('h1')).toBeTruthy();
  });
});
