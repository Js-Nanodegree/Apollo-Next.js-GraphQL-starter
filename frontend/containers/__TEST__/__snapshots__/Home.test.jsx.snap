// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home container should match snap shot 1`] = `
<div
  className="qepopj-0 gBRnsH"
>
  <h1>
    Apollo starter
  </h1>
</div>
`;
